code/game/objects/items/robot/items/gripper.dm

```diff
@@ -0,0 +1,218 @@
+		var/obj/machinery/cell_charger/cell_charger = target
+		if(cell_charger.charging)
+			gripped_item = cell_charger.charging
+			cell_charger.charging.add_fingerprint(user)
+			cell_charger.charging.forceMove(src)
+			cell_charger.removecell()
+
+	else if(ishuman(target))
+		var/mob/living/carbon/human/pickup_target = target
+		if(!IS_HORIZONTAL(pickup_target))
+			return FALSE
+		pickup_target.AdjustSleeping(-10 SECONDS)
+		pickup_target.AdjustParalysis(-6 SECONDS)
+		pickup_target.AdjustStunned(-6 SECONDS)
+		pickup_target.AdjustWeakened(-6 SECONDS)
+		pickup_target.AdjustKnockDown(-6 SECONDS)
+		pickup_target.stand_up()
+		playsound(user.loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
+		user.visible_message( \
+			"<span class='notice'>[user] shakes [pickup_target] trying to wake [pickup_target.p_them()] up!",\
+			"<span class='notice'>You shake [pickup_target] trying to wake [pickup_target.p_them()] up!",\
+			)
+
+	return TRUE
+
+
+// ===================
+// different grippers
+// ===================
+/obj/item/gripper/engineering
+	name = "engineering gripper"
+	desc = "A simple grasping tool for synthetic assets."
+	can_hold = list(
+		/obj/item/firealarm_electronics,
+		/obj/item/airalarm_electronics,
+		/obj/item/airlock_electronics,
+		/obj/item/firelock_electronics,
+		/obj/item/intercom_electronics,
+		/obj/item/apc_electronics,
+		/obj/item/tracker_electronics,
+		/obj/item/stock_parts,
+		/obj/item/vending_refill,
+		/obj/item/mounted/frame/light_fixture,
+		/obj/item/mounted/frame/apc_frame,
+		/obj/item/mounted/frame/alarm_frame,
+		/obj/item/mounted/frame/firealarm,
+		/obj/item/mounted/frame/display/newscaster_frame,
+		/obj/item/mounted/frame/intercom,
+		/obj/item/mounted/frame/extinguisher,
+		/obj/item/mounted/frame/light_switch,
+		/obj/item/assembly/prox_sensor,
+		/obj/item/rack_parts,
+		/obj/item/camera_assembly,
+		/obj/item/tank,
+		/obj/item/circuitboard,
+		/obj/item/stack/tile/light,
+		/obj/item/stack/ore/bluespace_crystal,
+		/obj/item/assembly/igniter,
+		/obj/item/flash,
+	)
+
+/obj/item/gripper/medical
+	name = "medical gripper"
+	desc = "A grasping tool used to hold organs and help patients up once surgery is complete."
+	can_hold = list(/obj/item/organ,
+					/obj/item/reagent_containers/iv_bag,
+					/obj/item/robot_parts,
+					/obj/item/stack/sheet/mineral/plasma, // for repair plasmamans
+					/obj/item/mmi,
+					/obj/item/reagent_containers/pill,
+					/obj/item/reagent_containers/drinks,
+					/obj/item/reagent_containers/glass,
+					/obj/item/reagent_containers/syringe,
+					)
+
+/obj/item/gripper/service
+	name = "Card gripper"
+	desc = "A grasping tool used to take IDs for paying taxes and waking up drunken crewmates"
+	can_hold = list(/obj/item/card,
+					/obj/item/camera_film,
+					/obj/item/paper,
+					/obj/item/photo,
+					/obj/item/toy/plushie,
+					/obj/item/disk/data,
+					/obj/item/disk/design_disk,
+					/obj/item/disk/plantgene,
```

Comment:
why does the service borg need the ability to pickup disks? this goes for all the other subtypes..

Reply:
They are service providers and always have been. For example, they want to help and teach nerds how to work with disks. Or drag the disk into cargo. They still cannot grow plants and will not be able to make a dangerous plant because of this.